src/app/integrations/intacct/intacct-shared/intacct-advanced-settings/intacct-advanced-settings.component.ts

```
@@ -220,6 +227,10 @@
     return paymentSync;
   }
 
+  isSingleCreditLineJEFieldVisible(): boolean {
+    return (this.reimbursableExpense === IntacctReimbursableExpensesObject.JOURNAL_ENTRY && this.reimbursableExportGroup == ExpenseGroupingFieldOption.CLAIM_NUMBER) || (this.corporateCreditCardExpense === IntacctCorporateCreditCardExpensesObject.JOURNAL_ENTRY && this.cccExportGroup == ExpenseGroupingFieldOption.CLAIM_NUMBER);
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)